lib/batch_sanitizer.js

```diff
@@ -0,0 +1,41 @@
+var fs = require("fs");
+
+var MINIFIED_AVG_LINE_LENGTH_CUTOFF = 100;
+
+function BatchSanitizer(files, stderr) {
+  this.files = files;
+  this.stderr = stderr || process.stderr;
+}
+
+BatchSanitizer.prototype.sanitizedFiles = function() {
+  var sanitizedFiles = [];
+
+  for(var i = 0; i < this.files.length; i++) {
+    if (this.isMinified(this.files[i])) {
+      this.stderr.write("WARN: Skipping " + this.files[i] + ": it appears to be minified\n");
```

Comment:
Yeah, I think this will be something to think about if we do start seeing false positives. Either tweaking the heuristic or adding some "no really I mean it" config flag. I figure we'll cross that bridge when we get to it. In the things we tested while developing it, no "real" code  really naturally came anywhere close to crossing this threshold.